src/api/index.js

```diff
@@ -2,69 +2,38 @@ import apiKey from './movie_api_key'
 
 // const dbUrl = 'http://localhost:3000/api/'
 const dbUrl = 'https://falsemotive.io/api/'
-const moviesUrl = 'https://api.themoviedb.org/3/movie/now_playing'
-
-// const options 
-
-export const postData = async (path = '', user) => {
-  const response = await fetch(`${dbUrl}${path}`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(user)
-  })
-
-  if (response.ok) {
-    return response;
-  } else {
-    throw new Error(`${response.statusText}`);
-  }
+const moviesUrl = 'https://api.themoviedb.org/3/movie/'
+const options = (method, data) => ({
```

Comment:
This is a great approach! Nice thinking. It's great to have these api functions more dynamic.